pkgs/development/compilers/llvm/default.nix

```diff
@@ -7,6 +7,7 @@
   stdenv,
   gcc12Stdenv,
   pkgs,
+  recurseIntoAttrs,
```

Comment:
You can also reference this as [lib.recurseIntoAttrs](https://noogle.dev/f/lib/recurseIntoAttrs).